core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -6352,6 +6352,20 @@ private void checkLiteral2(String expression, String expected) {
         .withCalcite().ok(expectedCalciteX);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-5265">[CALCITE-5265]
+   * Select operator' parentheses should be same with Union operator</a>. */
+  @Test void testInsertValueWithDynamicParams() {
```

Comment:
Are there any tests which cover what Julian has mentioned in the JIRA comment?

Reply:
There are many unit tests cover "insert into aTable select ? from bTable" in SqlParserTest.java. And RelToSqlConverterTest::testInsertValuesWithDynamicParams will cover "insert into aTable select ? from bTable union all select ? from cTable" mentioned by Julian in JIRA comment.